modules/onerepo/src/types.ts

```diff
@@ -56,29 +58,29 @@ export type Config = {
 	 * @default `/(\/__\w+__\/|\.test\.|\.spec\.|\.config\.)/`
 	 */
 	ignoreCommands?: RegExp;
-	/**
-	 * Give your CLI a unique name that's short and easy to remember.
-	 * If not provided, will default to `one`. That's great, but will cause conflicts if you try to use multiple monorepos that are both using oneRepo. But then again, what's the point of having multiple monorepos. Isn't that a bit besides the point?
-	 * @default `'one'`
-	 */
-	name?: string;
 	/**
 	 * Add shared commands and extra handlers. See the [official plugin list](https://onerepo.tools/docs/plugins/) for more information.
 	 * @default `[]`
 	 */
 	plugins?: Array<Plugin>;
 	/**
-	 * Absolute path location to the root of the repository.
-	 * @default `process.cwd()`
+	 * Must be set to `true` in order to denote that this is the root of the repository.
 	 */
-	root?: string;
+	root: true;
```

Comment:
is this not a boolean? or is it either true or not included at all?

Reply:
It's a bit hard to see in the diff here:

```
export type Config = RootConfig | WorkspaceConfig;
```

`root: true` denotes this is a `RootConfig` and `WorkspaceConfig` must not define `root` at all. That helps typescript determine which type of config it's using.